views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss

```diff
@@ -120,14 +126,14 @@ $modalShadowColor: #666666;
       font-style: normal;
       text-align: center;
       height: auto;
+      margin: 10px 10px 10px 0;
 
-      @include ltr {
-        margin: 10px 10px 10px 0;
-      }
       @include rtl {
-        margin: 10px 0 10px 10px;
+        display: flex;
```

Comment:
Did you try to move the `display: flex` rule to the `.audiorec-control` parent? 
This change should avoid the size jump when we switch the direction.

Reply:
parent already has display: flex, applying it here acts over the text and the svg image, if removed those two don't flip... but you're right there is a small jump in size, need to look into that...